README.md

```diff
@@ -72,10 +82,10 @@ When done, execute this command to download the dependency:
 $ helm dependency update
 ```
 
-Configure the template accordingly using the chart name as root element:
+Edit the `rest-http/values.yaml` file to configure the dependency `spring-boot-example-app`:
```

Comment:
Next, edit the `rest-http/values.yaml` file to configure the dependency to be used such as `spring-boot-example-app`:

Reply:
done